src/base/static/utils/misc-utils.js

```diff
@@ -0,0 +1,5 @@
+import modernizr from "../../../../.modernizrrc.js";
+
+const isTouchDevice = () => modernizr.touchevents;
```

Comment:
Can we use a singleton pattern here? It would save us the extra function call:

```javascript
const IS_TOUCH_DEVICE = modernizr.touchevents()
 export { IS_TOUCH_DEVICE }; 
```